tests/ui/shared/FailureSummaryTab_test.jsx

```diff
@@ -32,18 +42,26 @@ describe('FailureSummaryTab', () => {
   afterEach(() => {
     cleanup();
     fetchMock.reset();
+    const { pinnedJobs } = getState();
+    pinnedJobs.pinnedJobBugs.clear();
+    pinnedJobs.pinnedJobs = {};
   });
 
   const testFailureSummaryTab = () => (
     <Provider store={store}>
       <ConnectedRouter history={history}>
+        <PinBoard
+          classificationTypes={[{ id: 0, name: 'intermittent' }]}
+          isLoggedIn={false}
+          currentRepo={repositories[0]}
+        />
         <FailureSummaryTab
           selectedJob={selectedJob}
           jobLogUrls={jobLogUrls}
           logParseStatus="parsed"
           reftestUrl="boo"
           logViewerFullUrl="ber/baz"
-          addBug={() => {}}
+          addBug={(bug, job) => addBug(bug, job)(dispatch, getState)}
```

Comment:
Looks like this suggestion solved the redux issue? :) wondering if it'd be worth adding a comment to explain why you're doing this.